<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frmLogin</class>
 <widget class="QWidget" name="frmLogin">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>293</width>
    <height>142</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="maximumSize">
   <size>
    <width>293</width>
    <height>306</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Login</string>
  </property>
  <property name="windowIcon">
   <iconset resource="app_new.qrc">
    <normaloff>:/app_new/medfisis.ico</normaloff>:/app_new/medfisis.ico</iconset>
  </property>
  <property name="toolTip">
   <string>Login Dialog: check the user tooltip to have more information about this user.</string>
  </property>
  <property name="whatsThis">
   <string>This is a login dialog. Check the user tooltip to have more information about this user.</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,1">
     <item>
      <widget class="QLabel" name="lbUser">
       <property name="text">
        <string>Username</string>
       </property>
       <property name="buddy">
        <cstring>cmbUser</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cmbUser"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,1">
     <item>
      <widget class="QLabel" name="lbPasswd">
       <property name="text">
        <string>Password</string>
       </property>
       <property name="buddy">
        <cstring>linePasswd</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="linePasswd">
       <property name="autoFillBackground">
        <bool>true</bool>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="pushLogin">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="autoFillBackground">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>Login</string>
     </property>
     <property name="icon">
      <iconset resource="app_new.qrc">
       <normaloff>:/app_new/password.png</normaloff>:/app_new/password.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>16</width>
       <height>16</height>
      </size>
     </property>
     <property name="shortcut">
      <string>Ctrl+L</string>
     </property>
     <property name="autoDefault">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="1,0">
     <item>
      <widget class="QLabel" name="lbLocale">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>20</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>20</height>
        </size>
       </property>
       <property name="text">
        <string>Locale: Default</string>
       </property>
       <property name="scaledContents">
        <bool>false</bool>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>41</width>
         <height>31</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="app_new.qrc">:/app_new/edu_languages.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="app_new.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pushLogin</sender>
   <signal>clicked()</signal>
   <receiver>frmLogin</receiver>
   <slot>validate()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>82</x>
     <y>101</y>
    </hint>
    <hint type="destinationlabel">
     <x>146</x>
     <y>52</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cmbUser</sender>
   <signal>currentIndexChanged(QString)</signal>
   <receiver>frmLogin</receiver>
   <slot>updateTooltip(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>142</x>
     <y>33</y>
    </hint>
    <hint type="destinationlabel">
     <x>146</x>
     <y>67</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>validate()</slot>
  <slot>itemSelected(QString)</slot>
  <slot>loadForm(QString)</slot>
  <slot>showForm(QString)</slot>
  <slot>updateTooltip(QString)</slot>
 </slots>
</ui>
